src/internal/routineFunc.ts

```diff
@@ -121,3 +122,8 @@ export interface RoutineFunc {
     ...fns: OperatorFunction<any, any>[]
   ): Routine<unknown>;
 }
+
+export type SubscribeRoutineFunc = (
```

Comment:
Is there a specific reason this should be a separate type, and not just be in-lined in the method in `routinesRegistry`?